### ISSUE TITLE
IndexError for "*en" words

### ISSUE THREAD

**Issue by jirmauritz** [KeesCBakker/dutch-pluralizer-py#1]
Hi, thanks for this library!
I found a bug, where the singularize('wen') function fails with:
```
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-30-c22a683fcf9c> in <module>()
----> 1 singularize('wen')

3 frames
/usr/local/lib/python3.6/dist-packages/dutch_pluralizer/strategies/pluralize_with_en.py in singularize_with_en_double_consonant(plural)
     83     singular = plural[0:-2]
     84     if singular[-1] not in VOWELS and \
---> 85         singular[-1] == singular[-2] and \
     86         singular[-3] in VOWELS:
     87         return singular[0:-1]

IndexError: string index out of range
```

It happens for all kinds of three letter words that end with 'en'.

**Comment by KeesCBakker** [comment]
Sorry, this issue slipped my attention. I'll look into it and report back.

**Comment by KeesCBakker** [comment]
Fixed in version 0.0.36. Thanks for reporting. 

https://pypi.org/project/dutch-pluralizer/0.0.36/
